### ISSUE TITLE
GcStress: Linux: Crash in dbgjumper tests

### ISSUE THREAD

**Issue by swaroop-sridhar** [dotnet/coreclr#2854]
```
FAILED   - JIT/Methodical/VT/callconv/_il_dbgjumper3/_il_dbgjumper3.sh
               BEGIN EXECUTION
               /home/swaroops/test/Tests/coreoverlay/corerun _il_dbgjumper3.exe
               ./_il_dbgjumper3.sh: line 55:  6675 Segmentation fault      $_DebuggerFullPath "$CORE_ROOT/corerun" _il_dbgjumper3.exe $CLRTestExecutionArguments $Host_Args
               Expected: 100
               Actual: 139
               END EXECUTION - FAILED
```

```
FAILED   - JIT/Methodical/VT/callconv/_il_dbgjumper4/_il_dbgjumper4.sh
               BEGIN EXECUTION
               /home/swaroops/test/Tests/coreoverlay/corerun _il_dbgjumper4.exe
               ./_il_dbgjumper4.sh: line 55:  6701 Segmentation fault      $_DebuggerFullPath "$CORE_ROOT/corerun" _il_dbgjumper4.exe $CLRTestExecutionArguments $Host_Args
               Expected: 100
               Actual: 139
               END EXECUTION - FAILED
```

Here is another jumper test failing for the same reason
JIT/Methodical/VT/callconv/_dbgjumper/_dbgjumper

**Comment by sivarv** [comment]
This seems to be fixed with the current set of fixes that we have. Assigning to Lubo to verify and close.

**Comment by sivarv** [comment]
This test doesn't use mkrefany/refany val opcodes. Gc-hole Fixes in genPutStructArg() and genConsumePutStructArgStk() are fixing this test case.

**Comment by LLITCHEV** [comment]
Fixed by PR #3351 .
